src/perception/rai_whatisee/src/rai_whatisee_node.cpp

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if defined(ROS_DISTRO_HUMBLE)
```

Comment:
There could be a single flag to distinguish between Humble and all the later distros, for this purpose.
If we want it to be generic and future proof, we need to generate this flag from distro name, but it will be a moving target so this is ok.